meteorite_fall.js

```diff
@@ -49,6 +54,25 @@ function initializeMeteorites() {
     }
 }
 
+function checkGameOver(meteoriteX, meteoriteY, playerX, playerY, meteoriteWidth, meteoriteHeight, playerWidth) {
+    if (meteoriteY + meteoriteHeight == playerY) {
+        if (meteoriteX <= playerX + playerWidth && meteoriteX + meteoriteWidth >= playerX) {
+            isGameOver = true;
+            context.clearRect(0,0, canvasWidth, canvasHeight);
+            resetMeteorites();
+            alert("Game Over!");
+        }
+    }
+}
+
+function resetMeteorites() {
+    for (arrayIndex = 0; arrayIndex < meteorites.length; arrayIndex++) {
+        meteorites[arrayIndex].isFalling = false;
+        meteorites[arrayIndex].y = 0;
+    }
+}
+
+
```

Comment:
I dont like how there two empty lines next to one another

Reply:
ACK. I removed one.